### PR TITLE
Rewording`client` to `application`

### PR DESCRIPTION
**Pull request by luisrudge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -376,11 +376,11 @@ To get started you can just create a `WebAuth` instance like this
 ```js
 var auth0 = new auth0.WebAuth({
   domain: "{YOUR_AUTH0_DOMAIN}",
-  clientID: "{YOUR_AUTH0_CLIENT_ID}"
+  clientID: "{YOUR_AUTH0_APPLICATION_ID}"
```
Comment (cocojoe) [inline CHANGELOG.md]: this has not changed

File: README.md [hunk 2]
```diff
@@ -40,14 +40,14 @@ Provides support for all the authentication flows
 ```js
 var auth0 = new auth0.WebAuth({
   domain: "{YOUR_AUTH0_DOMAIN}",
-  clientID: "{YOUR_AUTH0_CLIENT_ID}"
+  clientID: "{YOUR_AUTH0_APPLICATION_ID}"
```
Comment (cocojoe) [inline README.md]: not changed

File: README.md [hunk 3]
```diff
@@ -40,14 +40,14 @@ Provides support for all the authentication flows
...
 });
 ```
 
 Parameters:
 - **domain {REQUIRED, string}**: Your Auth0 account domain such as `'example.auth0.com'` or `'example.eu.auth0.com'`.
-- **clientID {REQUIRED, string}**: Your Auth0 client ID.
-- **redirectUri {OPTIONAL, string}**: The URL where Auth0 will call back to with the result of a successful or failed authentication. It must be whitelisted in the "Allowed Callback URLs" in your Auth0 client's settings.
+- **clientID {REQUIRED, string}**: Your Auth0 application ID.
```
Comment (cocojoe) [inline README.md]: not changed or say Application's Client ID

File: README.md [hunk 4]
```diff
@@ -135,7 +135,7 @@ Provides an API client for the Auth0 Authentication API.
 ```js
 var auth0 = new auth0.Authentication({
   domain: "{YOUR_AUTH0_DOMAIN}",
-  clientID: "{YOUR_AUTH0_CLIENT_ID}"
+  clientID: "{YOUR_AUTH0_APPLICATION_ID}"
```
Comment (cocojoe) [inline README.md]: again

File: src/authentication/index.js [hunk 5]
```diff
@@ -14,14 +14,14 @@ var PasswordlessAuthentication = require('./passwordless-authentication');
 var DBConnection = require('./db-connection');
 
 /**
- * Creates a new Auth0 Authentication API client
+ * Creates a new Auth0 Authentication API application
  * @constructor
  * @param {Object} options
  * @param {String} options.domain your Auth0 domain
- * @param {String} options.clientID your Auth0 client identifier obtained when creating the client in the Auth0 Dashboard
+ * @param {String} options.clientID your Auth0 Application identifier obtained when creating the application in the Auth0 Dashboard
```
Comment (cocojoe) [inline src/authentication/index.js]: So shouldn't this be Application's client identifier? to be clear

Comment (cocojoe) [reply src/authentication/index.js]: There are more like this

**Comment by luisrudge** [comment]
I'll start over.

**Pull request closed without merging**